state/executor.go

```diff
@@ -80,7 +74,7 @@ func (e *BlockExecutor) InitChain(genesis *tmtypes.GenesisDoc) (*abci.ResponseIn
 				AppVersion: params.Version.AppVersion,
 			},
 		},
-		Validators:    tmtypes.TM2PB.ValidatorUpdates(tmtypes.NewValidatorSet(validators)),
+		Validators:    tmtypes.TM2PB.ValidatorUpdates(tmtypes.NewValidatorSet(nil)),
```

Comment:
why are we through the app passing nil validator to the abci app vs passing the active sequencer as the validator?
i.e the validator_set is solely determined by dymint but still being passed to the app. 

How does the app know how to distribute rewards for relevant sequencers?

Reply:
Currently, the app doesn't handle this
The sequencer on the rollapp is registered by tx / genesis for rewards distribution.